### DISCUSSION TITLE
New Changelog.md doesn't include changes from the latest release

### DISCUSSION THREAD

**Discussion by dooman87** [ImageMagick/ImageMagick#4976] [Development]
Hi there,

A new format of the changelog is nice and I guess it's populated automatically which eliminates possibility of missing changes. However, it seems to be missing the latest release. 

For instance, the current release is 7.1.0-28, but the latest I see in the changelog is 7.1.0-27. Am I missing something or it's known issue?

**Comment by urban-warrior** [comment]
Will need to investigate.  The auto changelog is not picking up the latest release tag for some reason.  Odd.

**Comment by dooman87** [comment]
@urban-warrior Do you know if changelog was moved to some other path? Can't find it in the last 2 releases.

**Reply by fmw42** [reply]
https://github.com/ImageMagick/Website/blob/main/ChangeLog.md

**Reply by dooman87** [reply]
Thank you!
